plugins/main/public/components/overview/github/panel/config/drilldown-action.tsx

```diff
@@ -176,15 +176,14 @@ export const DrilldownConfigAction = (drilldownProps: ModuleConfigProps) => {
                 {
                   id: 'rule.id', render: value => (
                     <EuiLink
-                      onClick={e =>
-                        AppNavigate.navigateToModule(e, 'manager', {
-                          tab: 'rules',
-                          redirectRule: value,
-                        })
-                      }
+                      onClick={e => {
+                        getCore().application.navigateToApp(rules.id, {
+                          path: `#/manager/?tab=rules&redirectRule=${value}`,
+                        });
+                      }}
                     >
                       {value}
-                    </EuiLink>
+                    </EuiLink >
```

Comment:
Maybe these redirections can be done with the component that opensearch offers to show in the bottom corner to which link the link redirects to.

```
<RedirectAppLinks application={getCore().application}>
 <EuiLink
   href={`${rules.id}#/manager/?tab=rules&redirectRule=${value}`}
 >
    {value}
  </EuiLink>
</RedirectAppLinks>

```

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6528/commits/bd9097d22abcd6af60ad48db3530eaa252f080eb